docs/monitors/kubernetes_monitor.md

```diff
@@ -30,6 +30,26 @@ fields behave differently when configured via k8s annotations:
 * lineGroupers (not supported at all)
 * path (the path is always fixed for k8s container logs)
 
+### Configuring multiple accounts per container
+
+Containers can be assigned one or more Scalyr API keys by specifying the `log.config.scalyr.com/{container_name}.teams.{team_number}.secret` annotation.  
+The `container_name` is the name of the container as specified in the pod spec, and the `team_number` is a unique number for each team.  
+The `secret` is the name of the secret holding the Scalyr API key for the team.  
+A default fallback api key can be specified by using the `log.config.scalyr.com/teams.{team_number}.secret` annotation.
+The monitor first looks into the container specific annotation, and if not found, it falls back to the default annotation or the global configuration api key.
```

Comment:
Was this extra default a requirement?  To me it makes more sense to just use the global write key as the default.

If this is retained, why use `log.config.scalyr.com/teams.{team_number}.secret` as the name?  Why not just `log.config.scalyr.com/teams.secret`?

Reply:
Yes, this was a requirement. And {team_number} is there to support multiple default keys. 
The way I understand it is that you can have default keys specifically for a single pod. But I don't know the use case.